Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -320,6 +326,37 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
     }
   };
 
+  publicAPI.resetXRScene = () => {
+    // Adjust world-to-physical parameters for different modalities
+
+    let distToTranslateZ = 0.0;
+    let rescaleFactor = 0.0;
+
+    if (model.xrSessionIsAR) {
+      // Scale down + translate forward in AR
+      rescaleFactor = 0.25;
+      distToTranslateZ = -0.5;
+    } else {
+      // Translate without scaling in VR
+      rescaleFactor = 1.0;
+      distToTranslateZ = -1.0;
```

Comment:
Based on testing, initializing the scene at 1m in front of the user is ok for seated VR experiences but potentially out of bounds for a standing VR walk-around experience. Need to experiment more with different XR reference spaces before settling on init factors.